site/components/TimeSelector.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { ReactElement } from 'react';
+import { ErrorInfo } from '../interfaces';
+import FormElementWrapper from './FormElementWrapper';
+
+interface TimeSelectorProps {
+    inputs?: {
+        hoursInput: string;
+        minuteInput: string;
+    };
+    errors?: ErrorInfo[];
+    startOrEnd: 'start' | 'end';
+}
+
+const TimeSelector = ({ inputs, startOrEnd, errors = [] }: TimeSelectorProps): ReactElement => (
+        <FormElementWrapper errors={errors} errorId={`${startOrEnd}-day-input`} errorClass="govuk-date-input--error">
+            <div className="govuk-date-input" id={`${startOrEnd}-date`}>
+                <div className="govuk-date-input__item">
+                    <div className="govuk-form-group">
+                        <label className="govuk-label govuk-date-input__label" htmlFor={`${startOrEnd}-hour-input`}>
+                            Hours
+                        </label>
+                        <input
+                            className={`govuk-input govuk-date-input__input govuk-input--width-2 ${
+                                errors.length > 0 ? 'govuk-input--error' : ''
+                            } `}
+                            id={`${startOrEnd}-hour-input`}
+                            name={`${startOrEnd}Hours`}
```

Comment:
the below input has a name of name={`${startOrEnd}Minute`} , but this one has 'Hours' - it would make sense to have them both be plural or neither plural, i.e. Minutes and Hours, or Minute and Hour

Reply:
done on my branch @alvak1  you can pull those changes